package.json

```diff
@@ -41,9 +41,8 @@
     "build:modules": "rollup -c rollup.config.js",
     "ci": "concurrently --names 'lint,test' 'npm run lint' 'npm run test'",
     "prepublishOnly": "npm run build",
-    "lint": "concurrently --names 'pretty,tslint' 'npm run pretty' 'npm run tslint'",
```

Comment:
No more lint?

Reply:
So tslint as a project is EOL'd, but lemme add eslint with the ts plugins.